monai/bundle/scripts.py

```diff
@@ -1539,3 +1539,126 @@ def init_bundle(
         copyfile(str(ckpt_file), str(models_dir / "model.pt"))
     elif network is not None:
         save_state(network, str(models_dir / "model.pt"))
+
+
+def _find_bundle_file(root_dir: Path, file_name: str, suffix: Sequence[str] = ("json", "yaml", "yml")) -> str | None:
+    # find bundle file with possible suffix
+    for _suffix in suffix:
+        full_name = f"{file_name}.{_suffix}"
+        if full_name in os.listdir(root_dir):
```

Comment:
this could be implemented with `os.path.isfile`, the function name could be `_find_config_file`

Reply:
Thanks, fixed it in this [commit](https://github.com/Project-MONAI/MONAI/pull/6835/commits/dc8858b53ca5d33b423f7a6d74644362d1a085f5).